cassandane/tiny-tests/JMAPEmail/email_parse_inmemory_blob

```diff
@@ -0,0 +1,45 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_email_parse_inmemory_blob
+    :min_version_3_9 :needs_component_jmap
```

Comment:
Version 3.4 is also affected.

Reply:
@elliefm is 3.4 still supported? I tagged this to backport to 3.8.

Reply:
@rsto I've tagged it for backporting as far as 3.4, and I'll see if it cherry-picks cleanly next time I'm doing a 3.4 release.  I wouldn't do a new 3.4 release just for this fix, but if at some point I'm doing a new 3.4 release for some other reason, this might as well be fixed too.

Reply:
Why isn’t this copied to the 3.4, 3.6 and 3.8 branches now, irrespective of whether there will be a release?